src/Ormolu/Printer/Internal.hs

```diff
@@ -26,6 +26,7 @@ module Ormolu.Printer.Internal
     enterLayout,
     vlayout,
     getLayout,
+    setAnchor,
```

Comment:
Try to solve the issue differently without introducing new primitives. We should try to keep our vocabulary of combinators from exploding and try to work within the DSL that we have. Look at my comments in #712 they should suggest how this can be solved and what layout is desirable.

Reply:
I'm sorry for unclear communication from my side, this alignment feature is not what I had in mind. As a rule, we do not align anything in Ormolu, so `y` would never have to be aligned with `x` because if `x` moved `y` would also need to move—this is exactly what we try to avoid in order to help with minimization of diffs.

Reply:
No problem, we didn't really discover the case with multiple pattern binds at first. At least, it was a fun exercise to add this functionality in ormolu, and helped me to get a better understanding of how internals work